### DISCUSSION TITLE
Any plans to add AWS support?

### DISCUSSION THREAD

**Discussion by danielmee** [DirectoryLister/DirectoryLister#1122] [Ideas & Features]
I've just been told by my hosting provider that I can't have all these files on a shared server and must move them to their AWS-style solution. So I went back to this software to see if I could use it to list objects in my bucket but doesn't look like it. Might be a totally different beast, I have no idea on these things, but just thought I'd raise it and see. Thanks.

**Comment by PHLAK** [comment]
Directory Lister is intended to be used for self-hosted files. If you need to list the contents of S3 buckets Directory Lister is not the software for you, sorry.
